cmd/backup-manager/app/clean/clean.go

```diff
@@ -60,16 +60,25 @@ func (bo *Options) cleanBRRemoteBackupData(ctx context.Context, backup *v1alpha1
 			return err
 		}
 
+		klog.Infof("For backup %s, try to delete %d objects", bo, len(objs))
+
 		// batch delete objects
 		result := s.BatchDeleteObjects(ctx, objs, opt.BatchDeleteOption)
 
 		if len(result.Deleted) != 0 {
-			klog.Infof("delete %d objects for cluster %s successfully: %s", len(result.Deleted), bo, strings.Join(result.Deleted, ","))
+			klog.Infof("For backup %s, delete %d objects successfully", bo, len(result.Deleted))
+			klog.V(4).Infof("For backup %s, all objects deleted: %s", bo, strings.Join(result.Deleted, ","))
 		}
-		if len(result.Errors) != 0 {
-			for _, oerr := range result.Errors {
-				klog.Errorf("delete object %s for cluster %s failed: %s", oerr.Key, bo, oerr.Err)
+		if len(result.Deleted) < len(objs) {
```

Comment:
It's possible that `len(result.Errors) != 0` when `len(result.Deleted) = len(objs)` from the code https://github.com/pingcap/tidb-operator/blob/v1.3.2/cmd/backup-manager/app/util/remote.go#L250.

Reply:
changed, both return error